src/tarski/reachability/gringo.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+import sys
+from clingo.application import Application, clingo_main
+
+"""
+A wrapper to emulate the default clingo app behavior
+"""
+class WrapperClingo(Application):
+    def __init__(self, name):
+        self.app_name = name
+
+    def main(self, ctl, files):
```

Comment:
Innocent question: do we need all this code if we're just simply using the default behavior of clingo?

Reply:
I had the same question. Going by the documentation this seems to be the recommended way to get the default clingo app behavior https://potassco.org/clingo/python-api/current/clingo/application.html 

Also, I checked it. The pip package doesn't include the gringo binary,  instead it has a platform specific library with a python wrapper. So, we cannot nab the path to the binary.